ui/ducks/send/helpers.js

```diff
@@ -72,14 +60,8 @@ export async function estimateGasLimitForSend({
       toAddress: to,
       amount: value,
     });
-
-    paramsForGasEstimate.to = sendToken.address;
   } else {
     if (!data) {
-      // eth.getCode will return the compiled smart contract code at the
```

Comment:
If we're leaving portions of the code in place, should we leave the comments also?

Gas is famously complex so no harm in more comments.